Database/Corrections/sodQuestFixes.lua

```diff
@@ -2286,6 +2286,66 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410023,
             [questKeys.zoneOrSort] = sortKeys.DRUID,
         },
+        [90133] = {
+            [questKeys.name] = "Quick Draw",
+            [questKeys.startedBy] = {nil,{208220}},
```

Comment:
The second entry of the `startedBy` table is a list object IDs, not item IDs, as you can see here: https://github.com/Questie/Questie/blob/6f0f1d61d4593447cbb40be01442141b4f4ef2fb/Database/questDB.lua#L5-L47

Even though **in theory** the "quest" is started by the items, it's still easier to simply add the NPCs which drop the map pieces directly. "item" quests are not even added to the map atm, only if we manually add a `creatureStart`.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5432/commits/4b6b39a0225ae357fa3b23cdf884a5bcf21336d4

Yes, we will need to wait until #5362 is finished before merging this.

Reply:
We can still merge it, but blacklist it for now, like we did with similar runes. There is a table in `SeasonOfDiscoveey.lua` for it.

Reply:
Also your commit didn't fix the initial issue.

You wrote item IDs in an object ID field. Not you are writing NPC IDs in an object field.